src/Io/functions.php

```diff
@@ -29,6 +29,11 @@ function fds($path = '/dev/fd')
             }
         }
     }
+     else {
+         foreach ($fds as $i => $fd) {
+             $fds[$i] = (Int) $fd;
```

Comment:
nit: CS

Changes LGTM otherwise, perhaps adjust return type to `int[]` accordingly?

Reply:
Changed it :+1: